.github/workflows/superset-backend.yml

```diff
@@ -114,14 +109,9 @@ jobs:
       REDIS_PORT: 16379
     services:
       mysql:
-        image: mysql:5.7
+        image: yobasystems/alpine-mariadb:10.4.12
```

Comment:
This saves us another 10 seconds and now the full CI process can finish in under 7 minutes (with cache)!